content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,212 @@
+
+When the above program is executed, we get the output below:
+```Matlab
+f =
+ 
+2/(1 + w*2i) - (2*limit(exp(-t/2)*exp(-t*w*1i), t, Inf))/(1 + w*2i)
+```
+For formatting of this output to a user-friendly manner, we use a function pretty(). `pretty(f)` prints the symbolic expression `f` in a format that resembles type-set mathematical equations.
+```matlab
+ pretty(f)
+           /             /   t \              \
+           |    lim   exp| - - | exp(-t w 1i) | 2
+    2      \ t -> Inf    \   2 /              /
+-------- - --------------------------------------
+1 + w 2i                  1 + w 2i
+```
+Now, we need to get the values of `w` since we use them to plot the Fourier transform. To get the values, we execute the code below:
+```Matlab
+f_sub = subs(f, w, [-pi: pi])
+```
+`subs` mean symbolic substitution. It means all the symbolic variables are substituted. This function replaces the symbolic variable in `f` with the values of `w`, which is in the range of `-pi` to `pi`. When we do this, we get the values below:
+```Matlab
+f_sub =
+ 
+[ -2/(- 1 + pi*2i), 39614081257132168796771975168/383180597959374460738291811785 + 169674450797966737934326431744i/383180597959374460738291811785, 39614081257132168796771975168/123059858877705322463182898633 + 90446288283702400340782481408i/123059858877705322463182898633, 39614081257132168796771975168/21395444824564859375161886153 + 11218125769438062747238531072i/21395444824564859375161886153, 39614081257132168796771975168/78187355799953071474228774345 - 68010036744826274846305419264i/78187355799953071474228774345, 39614081257132168796771975168/293435591803869958760383563209 - 147238199259090612439849369600i/293435591803869958760383563209, 39614081257132168796771975168/667140152836315521233626252745 - 226466361773354950033393319936i/667140152836315521233626252745]
+```
+Now, we plot these values using the `plot()` function.
+```Matlab
+subplot(2, 1, 2);
+plot(angle(double(f_sub)));   %angle are the w values.
+title('phase spectrum')   % title
+```
+`plot(angle(double(data_value)))` gives the phase spectrum plot. The function `subplot()` is for making a subplot. The `title()` gives the plot a title.  
+
+![phase spectrum](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-three.png)
+*Phase spectrum plot*
+
+Now lets plot the magnitude response using the same values of `w`.
+```Matlab
+subplot(2, 1, 1); 
+plot(abs(double(f_sub))); %magnitude spectrum plot
+title('magnitude spectrum')
+```
+The code `plot(abs(double(data_values)))` gives the magnitude spectrum. This plot uses the absolute values of the data, thus `abs()`.
+
+![magnitude spectrum](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-four.png)
+*Magnitude spectrum plot*
```

Comment:
Then for all the image descriptions, add a linebreak(space) in between the image and the description because in the preview, they render on the same line.